app/workers/publishing_api_document_republishing_worker.rb

```diff
@@ -78,11 +76,7 @@ def the_document_has_an_edition_to_check?
   end
 
   def the_document_has_been_unpublished?
-    pre_publication_edition && pre_publication_edition.unpublishing
-  end
-
-  def the_document_has_been_withdrawn?
-    live_edition && live_edition.unpublishing
+    live_edition && (live_edition.unpublished? || live_edition.withdrawn?)
```

Comment:
We talked about:

I don't think it's safe to consolidate the `the_document_has_been_unpublished?` and the `the_document_has_been_withdrawn?` methods because `live_edition` has to be publicly visible – which means it can only ever be `published` or `withdrawn`.

Unpublished editions are not publicly visible, so are not 'live' per se.

Reply:
I've reverted back to the old implementation until we figure out a way to make this work. It's not currently working for unpublished editions

Reply:
We had to check for an unpublished edition separately in the end. The behaviour for withdrawn and unpublished documents remains separate